### PR TITLE
ADM-378 add verification for the next page

### PR DESCRIPTION
**Pull request by fengxinhou**
## Summary

Add verification for the next page

## Before

Next button is enable when projectName、startDate、endDate is empty
configdata empty

## After

After input projectName、startDate、endDate, and all boards selected verified successfully, next button is enabled

### CODE REVIEW

File: frontend/__tests__/src/components/Metrics/MetricsStepper/MetricsStepper.test.tsx [hunk 1]
```diff
@@ -114,17 +124,35 @@ describe('MetricsStepper', () => {
...
   })
 
+  it('should disable next when dataRange is empty ', async () => {
+    const { getByText, getByRole } = setup()
+    await fillConfigPageData()
+
+    const startDateInput = getByRole('textbox', { name: START_DATE_LABEL }) as HTMLInputElement
+    const endDateInput = getByRole('textbox', { name: END_DATE_LABEL }) as HTMLInputElement
+
+    fireEvent.change(startDateInput, { target: { value: null } })
```
Comment (guzhongren) [inline frontend/__tests__/src/components/Metrics/MetricsStepper/MetricsStepper.test.tsx:134]: use `userEvent`

File: frontend/src/components/Metrics/MetricsStepper/index.tsx [hunk 2]
```diff
@@ -37,7 +39,9 @@ const MetricsStepper = () => {
         { key: isShowSourceControl, value: isSourceControlVerified },
       ]
       const activeParams = showNextButtonParams.filter(({ key }) => key)
-      hasMetrics ? setIsDisableNextButton(!activeParams.every(({ value }) => value)) : setIsDisableNextButton(true)
+      projectName && dateRange.startDate && dateRange.endDate && hasMetrics
+        ? setIsDisableNextButton(!activeParams.every(({ value }) => value))
```
Comment (guzhongren) [inline frontend/src/components/Metrics/MetricsStepper/index.tsx:43]: `activeParams.every(({ value }) => value)`
what does mean?
https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array/every

File: frontend/__tests__/src/components/Metrics/MetricsStepper/MetricsStepper.test.tsx [hunk 3]
```diff
@@ -46,6 +48,23 @@ const setup = () =>
     </Provider>
   )
 
+const fillConfigPageData = async () => {
+  const input = screen.getByRole('textbox', { name: PROJECT_NAME_LABEL })
```
Comment (bhan-tw) [inline frontend/__tests__/src/components/Metrics/MetricsStepper/MetricsStepper.test.tsx:52]: Better to define an understandable field for `input `

**Review by bhan-tw — APPROVED** [review]
LGTM

File: frontend/src/components/Metrics/MetricsStepper/index.tsx (hunk 2 above)
Comment (fengxinhou) [reply frontend/src/components/Metrics/MetricsStepper/index.tsx:43]: First use the key value to find the displayed component, and then use the value of the object to determine whether the component was successfully validated, to determine whether next is enabled

**Review by bhan-tw — APPROVED** [review]

**Pull request merged**